### DISCUSSION TITLE
how do i exclude a specific heading?

### DISCUSSION THREAD

**Discussion by zoofys** [allejo/jekyll-toc#69] [Q&A] [answered]
how do i exclude a specific heading?

```
# Heading 1 <-- i want to exclude this
## Heading 2
# Heading 3  <-- i want to exclude this

```

**Comment by allejo** [comment, marked as answer]
Two ways of accomplishing this and it depends on the situation that you want.

1. If you want to ignore all headings that are level 1 (i.e. `<h1>` or `#`), you can make use of the `h_min` parameter; set it to 2 and it will only include headings level 2 and above (i.e. `<h2>` or `##`). You can also limit the other side with `h_max`, so say you want to ignore `<h6>` from the TOC.
2. If you want to exclude **ONLY** specific headings, you can attach the `.no_toc` class to the headings you want to exclude.
   ```markdown
   # Heading 1
   {:.no_toc}
   ```
